### PR TITLE
docs(storybook): configs new documentation structure

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-352
https://jirasoftware.catho.com.br/browse/QTM-353

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Code review
- [ ] Stories

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,19 +63,18 @@
...
+    "@storybook/addon-a11y": "^6.5.9",
+    "@storybook/addon-essentials": "^6.5.9",
+    "@storybook/addon-links": "^6.5.9",
+    "@storybook/addons": "^6.5.9",
+    "@storybook/api": "^6.5.9",
+    "@storybook/channels": "^6.5.9",
+    "@storybook/client-logger": "^6.5.9",
+    "@storybook/core-events": "^6.5.9",
+    "@storybook/react": "^6.5.9",
+    "@storybook/router": "^6.5.9",
```
Comment (MarcosViniciusPC) [inline package.json:75]: Is there a need to use this package in the project?

File: .storybook/regression-test/main.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
   stories: ['../../stories/**/*.regression-test.story.jsx'],
-  addons: ['@storybook/addon-backgrounds'],
+  addons: [
+    '@storybook/addon-backgrounds',
+    '@storybook/addon-viewport/register',
```
Comment (MarcosViniciusPC) [inline .storybook/regression-test/main.js:5]: The addon using @storybook/addon-viewport is a dependency of the 'essentials' package, was it intentional to use a dependency of a dependency in this case? Wouldn't it be better to use the 'essentials' package specifying which plugins to use similar to the documentation?

**Review by MarcosViniciusPC — DISMISSED** [review]
[x] Code review
[x] Stories

File: package.json [hunk 3]
```diff
@@ -63,19 +63,18 @@
...
-    "@storybook/storybook-deployer": "^2.8.10",
-    "@storybook/theming": "^6.4.14",
+    "@storybook/addon-a11y": "^6.5.9",
+    "@storybook/addon-essentials": "^6.5.9",
+    "@storybook/addon-links": "^6.5.9",
+    "@storybook/addons": "^6.5.9",
+    "@storybook/api": "^6.5.9",
+    "@storybook/channels": "^6.5.9",
+    "@storybook/client-logger": "^6.5.9",
+    "@storybook/core-events": "^6.5.9",
```
[the remark this answers is not in the document]
Comment (toncabral88) [reply package.json:73]: I'll check this

File: package.json (hunk 1 above)
Comment (toncabral88) [reply package.json:75]: I'll check this

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review
[x] Stories

**Pull request merged**